### PR TITLE
Set vhost-user-fs-pci device's bus to pcie root port

### PR DESCRIPTION
**Pull request by xiagao**
ID: 2023200
Signed-off-by: Xiaoling Gao <xiagao@redhat.com>

### CODE REVIEW

File: virttest/qemu_devices/qdevices.py [hunk 1]
```diff
@@ -323,8 +323,9 @@ def verify_unplug(self, out, monitor):      # pylint: disable=W0613,R0201
     def is_pcie_device(self):
         """Check is it a pcie device"""
         driver = self.get_param("driver", "")
-        if (driver in ("e1000e", "vhost-vsock-pci", "qemu-xhci", "vfio-pci")
-                or driver.startswith("virtio-")):
+        driver_list = ["e1000e", "vhost-vsock-pci", "qemu-xhci", "vfio-pci",
```
Comment (vivianQizhu) [inline virttest/qemu_devices/qdevices.py:326]: Recommend to name it `pcie_drivers`.

Comment (xiagao) [reply virttest/qemu_devices/qdevices.py]: Update.

**Review by vivianQizhu — APPROVED** [review]

**Comment by vivianQizhu** [comment]
@YongxueHong Request for your review, thanks a lot.

File: virttest/qemu_devices/qdevices.py [hunk 2]
```diff
@@ -323,8 +323,9 @@ def verify_unplug(self, out, monitor):      # pylint: disable=W0613,R0201
     def is_pcie_device(self):
         """Check is it a pcie device"""
         driver = self.get_param("driver", "")
-        if (driver in ("e1000e", "vhost-vsock-pci", "qemu-xhci", "vfio-pci")
-                or driver.startswith("virtio-")):
+        pcie_drivers = ["e1000e", "vhost-vsock-pci", "qemu-xhci", "vfio-pci",
+                        "vhost-user-fs-pci"]
+        if (driver in pcie_drivers or driver.startswith("virtio-")):
```
Comment (YongxueHong) [inline virttest/qemu_devices/qdevices.py:328]: LGTM.
BTW, we can improve the code:
```suggestion
        return (driver in pcie_drivers or driver.startswith("virtio-"))
```

Comment (xiagao) [reply virttest/qemu_devices/qdevices.py]: Update.

**Review by YongxueHong — APPROVED** [review]

**Comment by vivianQizhu** [comment]
@xiagao Would you please provide the test result somewhere? Thanks.

**Comment by xiagao** [comment]
(1/1) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.virtio_fs_share_data.with_cache.always.q35: PASS (89.12 s)

20:04:53 DEBUG| [stderr]     -chardev socket,id=char_virtiofs_fs,path=/tmp/avocado_eyff340p/avocado-vt-vm1-fs-virtiofsd.sock \

20:04:53 DEBUG| [stderr]     -device pcie-root-port,id=pcie-root-port-3,port=0x3,addr=0x1.0x3,bus=pcie.0,chassis=4 \

20:04:53 DEBUG| [stderr]     -device vhost-user-fs-pci,id=vufs_virtiofs_fs,chardev=char_virtiofs_fs,tag=myfs,queue-size=1024,bus=pcie-root-port-3,addr=0x0 \

**Pull request merged**